extensions/unsplash/src/images.tsx

```diff
@@ -0,0 +1,47 @@
+import { List, Icon } from "@raycast/api";
+
+// Hooks
+import { useSearch } from "./hooks/useSearch";
+
+// Components
+import Actions from "./components/Actions";
+
+// Types
+interface SearchListItemProps {
+  searchResult: SearchResult;
+}
+
+const Unsplash: React.FC = () => {
+  const { state, search } = useSearch("photos");
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search wallpapers..." throttle>
+      <List.Section title="Results" subtitle={String(state?.results?.length)}>
+        {state.results.map((result) => (
+          <SearchListItem key={result.id} searchResult={result} />
+        ))}
+      </List.Section>
+    </List>
+  );
+};
+
+const SearchListItem: React.FC<SearchListItemProps> = ({ searchResult }) => {
+  const [title, description, image] = [
+    searchResult.description || searchResult.alt_description || searchResult.user.name || "No Name",
+    searchResult.description ? searchResult.alt_description : undefined,
+    searchResult.urls?.thumb || searchResult.urls?.small || searchResult.urls?.regular,
+  ];
+
+  return (
+    <List.Item
+      title={title}
+      icon={image}
+      subtitle={description}
+      accessoryTitle={searchResult.user.name}
+      accessoryIcon={Icon.Person}
```

Comment:
You could do something like this (with fallback):

```accessoryIcon={{ source: searchResult.user.profile_image.small, mask: ImageMask.Circle }}```

Reply:
I did that at the beginning, though I didn't want to put it because downloading avatars doesn't really mean anything but more data usage, should I do it?

Reply:
That's fine... We handle this natively and cache it ;-)

Reply:
it looks nicer :-)

Reply:
That's 🔥